de-lib/src/main/java/org/iplantc/de/theme/base/client/admin/BelphegorDisplayStrings.properties

```diff
@@ -17,7 +17,7 @@ addCategoryPrompt = New Category
 appCategorizeSuccess = Successfully added {0} to the following Categories: {1,list}.
 appCategorizeSuccess[\=1] = Successfully added {0} to the following Category: {1,list}.
 confirmDeleteAppCategory = Do you want to delete the category \"{0}\"?
-confirmDeleteAppTitle = Delete App
+confirmDeprecateAppTitle = Delete App
```

Comment:
Hm... that's an interesting point.  I went through and updated the methods and properties for consistency between what's seen in the UI, but if it's clearer to leave method names as they were to match the service verbiage, then I'm ok with reverting.